app/cloudinit/cloudinit.go

```diff
@@ -402,6 +402,21 @@ else
     fi
 fi
 chmod 777 /usr/local/bin/plugin
+
+{{ if .Tmate.Enabled }}
```

Comment:
is this the only place we need to make the change? for both tart and anka?

Reply:
yes

Reply:
anka is irrelevant and this is the only script for mac. The other mac is irrelevant as we have only arm64 (tart support only apple silicon)